src/host/path_isabsolute.c

```diff
@@ -17,13 +17,59 @@ int path_isabsolute(lua_State* L)
 
 int do_isabsolute(const char* path)
 {
-	return (
-		path[0] == '/' ||
-		path[0] == '\\' ||
-		path[0] == '$' ||
-		path[0] == '%' ||
-		(path[0] == '"' && path[1] == '$') ||
-		(path[0] == '"' && path[1] == '%') ||
-		(path[0] != '\0' && path[1] == ':')
-	);
+	char c;
+	char* closing;
```

Comment:
Since this contains a pointer into the `path` string, shouldn't this also be `const`?